src/sentry/api/urls.py

```diff
@@ -2319,8 +2319,9 @@
     ),
     # Groups
     url(r"^(?:issues|groups)/", include(GROUP_URLS)),
+    # TODO: include in the /organizations/ route tree + remove old dupe once hybrid cloud launches
     url(
-        r"^issues/(?P<organization_slug>[^\/]+)/(?P<issue_id>[^\/]+)/participants/$",
+        r"^organizations/(?P<organization_slug>[^\/]+)/issues/(?P<issue_id>[^\/]+)/participants/$",
```

Comment:
Do we need the old URL for participants?

Reply:
i'm keeping it for now since it's getting some occasional hits